src/L.Control.Locate.js

```diff
@@ -37,19 +37,19 @@ const LocationMarker = Marker.extend({
     let style = "";
 
     if (opt.color !== undefined) {
-      style += `stroke:${opt.color};`;
+      style += ` stroke="${opt.color}"`;
```

Comment:
Instead of adding the space here, let's add the styles to an array and then join it on `" "`. That makes it easier to guarantee space between items.